wiki/Tournaments/MWC/2024_4K/en.md

```diff
@@ -443,7 +444,7 @@ Sunday, 25 August 2024:
 
 ### Qualifiers
 
-The final standings for the Qualifier stage can be found in the following [spreadsheet](https://docs.google.com/spreadsheets/d/1Al5x3RJQfKD9FiEV9lJAmX0S_ajMdndogNM7_quP3PM/edit?gid=754304763#gid=754304763).
+The final standings for the Qualifier stage can be found in the following [spreadsheet](https://docs.google.com/spreadsheets/d/1Al5x3RJQfKD9FiEV9lJAmX0S_ajMdndogNM7_quP3PM/edit?gid=754304763#gid=754304763).\
```

Comment:
we only stack hatnotes and notices without the paragraph gap.. ?

Reply:
That would be inconsistent with the VOD annotations for other stages (as originally suggested by you btw)

Reply:
the spreadsheet lines aren't worded like the kinds of notices we stack, they don't seem to be standardised, and they don't have the whole sentence linked like the other vod and mappack notices. similar to the [sign-up list mention](https://osu.ppy.sh/wiki/en/Tournaments/MWC/2024_4K#participants), these look like regular paragraphs

also notices are supposed to stick below the heading, above any regular paragraphs